websocket/frontend_server/CHANGELOG.md

```diff
@@ -0,0 +1,88 @@
+## v3.0.0
+
```

Comment:
What's this file? Is it relevant to this sample?

Reply:
it is irrelevant, I will remove it.